src/trace_processor/importers/proto/gpu_event_parser.cc

```diff
@@ -490,6 +491,15 @@ void GpuEventParser::ParseGpuRenderStageEvent(
               }
             }
           }
+
+          if (event.render_pass_instance_id()) {
+            base::StackString<512> id_str("rp:#%" PRIu64,
+                                          event.render_pass_instance_id());
+            inserter->AddArg(context_->storage->InternString("correlation_id"),
```

Comment:
Please intern this in the constructor to prevent repeated internstring on a constant.